kpconv_torch/datasets/S3DIS.py

```diff
@@ -29,19 +28,24 @@ def __init__(
         chosen_log=None,
         infered_file=None,
         output_dir=None,
-        split="training",
         use_potentials=True,
         load_data=True,
+        split="training",
     ):
+        PointCloudDataset.__init__(
+            self,
+            config=config,
+            datapath=datapath,
+            dataset="S3DIS",
+            chosen_log=chosen_log,
+            infered_file=infered_file,
+            output_dir=output_dir,
+            split=split,
+        )
         """
         This dataset is small enough to be stored in-memory, so load all point clouds here
         """
 
-        self.train_save_path = get_train_save_path(output_dir, chosen_log)
-        self.test_save_path = get_test_save_path(infered_file, chosen_log)
-
-        PointCloudDataset.__init__(self, "S3DIS")
-
```

Comment:
Same remarks:

- do not add code before docstring
- use `super().__init__(...)` to call the constructor of the parent class